spec/models/manageiq/providers/vmware/discovery_spec.rb

```diff
@@ -0,0 +1,112 @@
+describe ManageIQ::Providers::Vmware::Discovery do
+  describe ".probe" do
+    let(:discover_types) { %i(virtualcenter esx vmwareserver) }
+    let(:ost)            { OpenStruct.new(:ipaddr => "0.0.0.0", :discover_types => discover_types, :hypervisor => [], :os => []) }
+    before do
+      require "VMwareWebService/MiqVimClientBase"
+
+      miq_vim_client_base = double("VMwareWebService/MiqVimClientBase")
+      allow(miq_vim_client_base).to receive(:about).and_return(about_info)
```

Comment:
where is `about_info` defined?

Reply:
never mind.  I see it below.

Reply:
:+1: yeah about_info is what is different between the different contexts